geomstats/visualization/symmetric_positive_definite.py

```diff
@@ -0,0 +1,598 @@
+"""Functions for SPD Visualization."""
+
+import colorsys
+import math
+
+import matplotlib.pyplot as plt
+import numpy as np
+from geomstats.geometry.spd_matrices import SPDAffineMetric, SPDMatrices
+from geomstats.visualization import Ellipses
+from matplotlib.tri import Triangulation
+from mpl_toolkits.mplot3d.art3d import Poly3DCollection
+
+
+class SymPosDefVisualization:
+    """Class for the visualization of the manifold for SPD matrices.
+
+    This class provides all the essential methods for the
+    visualization of the manifold of the Symmetric Positive
+    Definite matrices.
+
+    Parameters
+    ----------
+    max_z: int
+    The scaling factor of the manifold
+
+    Attributes
+    ----------
+    currZ: int
+    The scaling factor of the manifold
+
+    spdPointViz:
```

Comment:
no camelcase in python, except for class names -- > rename all variables that have camelcase